docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md

```diff
@@ -150,6 +151,29 @@ Users can install and run multiple versions of the .NET Framework on their compu
 
     - It checks in order from most recent version to earliest version.
 
+<a name="ps_a"></a> 
+#### To check for a minimum-required .NET Framework version by querying the registry in PowerShell (.NET Framework 4.5 and later)
+
+- The following example checks the value of the `Release` keyword to determine whether .NET Framework 4.6.2 or higher is installed (returning `True` if it is and `False` otherwise). You can substitute another value listed in the table to check for a different minimum-required .NET Framework version.
+
+    |Version|Minimum value of the Release DWORD|
+    |-------------|--------------------------------|
+    |.NET Framework 4.5|378389|
+    |.NET Framework 4.5.1|378675|
+    |.NET Framework 4.5.2|379893|
+    |[!INCLUDE[net_v46](../../../includes/net-v46-md.md)] installed with Windows 10|393295|
+    |[!INCLUDE[net_v46](../../../includes/net-v46-md.md)] installed on all other Windows OS versions|393297|
+    |[!INCLUDE[net_v461](../../../includes/net-v461-md.md)] installed on Windows 10|394254|
+    |[!INCLUDE[net_v461](../../../includes/net-v461-md.md)] installed on all other Windows OS versions|394271|
+    |[!INCLUDE[net_v462](../../../includes/net-v462-md.md)] installed on Windows 10 Anniversary Update|394802|
+    |[!INCLUDE[net_v462](../../../includes/net-v462-md.md)] installed on all other Windows OS versions|394806|
+    |.NET Framework 4.7 installed on Windows 10 Creators Update|460798|
+    |.NET Framework 4.7 installed on all other Windows OS versions|460805|
+
+```PowerShell
```

Comment:
Wondering if we move the example up right after the sentence we introduce the example and then show the table with the different values you can use after the example.